packages/react-components/react-tree/src/components/Tree/index.ts

```diff
@@ -4,3 +4,4 @@ export * from './useTree';
 export * from './useTreeContextValues';
 export * from './useTreeStyles.styles';
 export * from './renderTree';
+export * from './Collapse';
```

Comment:
I'm not sure if this needs to be here; I was trying to resolve a compiler error.